### PR TITLE
Fixes #145 MJPEG on Safari OS X

### LINKED ISSUE #145: MJPEG on Safari OS X not working
https://github.com/AxisCommunications/media-stream-library-js/blob/1dcdad4de637a31291a4f7c6ab6de656a29a5475/lib/components/canvas/index.ts#L119

```
window.createImageBitmap is not a function. (In 'window.createImageBitmap(blob)', 'window.createImageBitmap' is undefined)
```

Maybe drawImage for Safari and transferFromImageBitmap for Chrome/Firefox? 

Both versions, 5 and 6

### PR DESCRIPTION
**Pull request by reijosirila**
Fixes issue  #145. 

Also discovered that getContext cannot be called twice on safari: 

Screenshot 2019-06-20 15 00 25

That's why fallback at the beginning didn't work.

### CODE REVIEW

File: lib/components/canvas/index.ts [hunk 1]
```diff
@@ -106,19 +106,20 @@ export class CanvasSink extends Sink {
...
-      // (https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/createImageBitmap)
-      // Note: drawImage can also be used instead of transferFromImageBitmap, but it caused
-      // very large memory use in Chrome (goes up to ~2-3GB, then drops again).
       drawImageBlob = ({ blob }) => {
         info.renderedFrames++
         window
           .createImageBitmap(blob)
           .then(imageBitmap => {
-            ;(ctx as any).transferFromImageBitmap(imageBitmap)
+            ; (ctx as any).transferFromImageBitmap(imageBitmap)
```
Comment (steabert) [inline lib/components/canvas/index.ts]: Did you run `prettier` on this file? We use `prettier` to handle any formatting so it stays consistent.

Comment (reijosirila) [reply lib/components/canvas/index.ts]: Well, I have plugins on editor that should follow tslint, eslint, editor and prettier rules.. Editor automatically updated this.. I'll run it later manually and update PR.

**Comment by reijosirila** [comment]
Runned pretteier, now is should be OK.

File: lib/components/canvas/index.ts [hunk 2]
```diff
@@ -165,15 +166,13 @@ export class CanvasSink extends Sink {
...
-          const jpegMedia = msg.sdp.media.find(
-            (media): media is VideoMedia => {
-              return (
-                media.type === 'video' &&
-                media.rtpmap !== undefined &&
-                media.rtpmap.encodingName === 'JPEG'
-              )
-            },
-          )
+          const jpegMedia = msg.sdp.media.find((media): media is VideoMedia => {
```
Comment (steabert) [inline lib/components/canvas/index.ts]: There must be still something odd with your prettier setup, this line should be split.

Comment (reijosirila) [reply lib/components/canvas/index.ts]: Reinstalled everyting using `--frozen-lockfile` and result is the same. 
`
reijo@rmackbook axis (safari_mjpeg) $ yarn prettier lib/components/canvas/index.ts | grep "const jpegMedia" -A 2 -B 2
          // Initialize first timestamp and clockrate
          firstTimestamp = 0
          const jpegMedia = msg.sdp.media.find((media): media is VideoMedia => {
            return (
              media.type === 'video' &&
`
Can you please push a commit how ever you like it ?

Comment (steabert) [reply lib/components/canvas/index.ts]: I'll have a look ASAP.

Comment (steabert) [reply lib/components/canvas/index.ts]: hmm, seemed to be wrong on master, so I'll just merge this. I'll add a test on master to check and reformat if necessary.

**Pull request merged**